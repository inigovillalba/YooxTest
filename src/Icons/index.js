export { default as Audio } from "./Audio/Audio";
export { default as Delivery } from "./Delivery/Delivery";
export { default as Edit } from "./Edit/Edit";
export { default as Favourites } from "./Favourites/Favourites";
export { default as GiftCards } from "./GiftCards/GiftCards";
