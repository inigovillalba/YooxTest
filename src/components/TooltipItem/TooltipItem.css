.tooltip-item {
  height: 36px;
  padding: 7px 10px 9px 10px;
  color: #ffffff;
  box-sizing: border-box;
}

.tooltip-item:hover {
  background-color: #4f4f4f;
  cursor: pointer;
}

.tooltip-item span {
  padding-left: 10px;
  vertical-align: super;
}
