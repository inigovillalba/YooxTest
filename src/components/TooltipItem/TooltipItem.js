import React, { Component } from "react";
import PropTypes from "prop-types";

import "./TooltipItem.css";

class TooltipItem extends Component {
  static propTypes = {
    children: PropTypes.arrayOf(PropTypes.element).isRequired
  };

  render() {
    return <li className="tooltip-item">{this.props.children}</li>;
  }
}

export default TooltipItem;
