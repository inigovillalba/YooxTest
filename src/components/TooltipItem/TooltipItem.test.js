/* eslint-env jest */
/* eslint-disable padded-blocks, no-unused-expressions */

import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import TooltipItem from "./TooltipItem";
import { Audio } from "../../Icons";

describe("TooltipItem", () => {
  test("renders correctly", () => {
    const renderer = new ShallowRenderer();

    const wrapper = renderer.render(
      <TooltipItem>
        <Audio />
        <span>Some Text</span>
      </TooltipItem>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
