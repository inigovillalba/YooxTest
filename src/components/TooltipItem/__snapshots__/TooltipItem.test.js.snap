// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TooltipItem renders correctly 1`] = `
<li
  className="tooltip-item"
>
  <Audio
    fillColor="#ffffff"
    height={20}
    width={20}
  />
  <span>
    Some Text
  </span>
</li>
`;
